### PR TITLE
Use MIT license

### PR DESCRIPTION
**Pull request by erikkemperman**

### CODE REVIEW

**Comment by erikkemperman** [comment]
Feels a bit weird to assert copyright on behalf of Microsoft Corp. but there's probably some history I don't know about. Done!

**Comment by dbrattli** [comment]
Yes, I had to sign an agreement with them back in 2013 in order to release the code, as I was MS employee at the time. The first release of RxPY was also released through Microsoft Open Technologies and I don't feel too comfortable ripping that out because we change the license to MIT. There's still a lot of code directly ported from RxJS and Rx.NET. Ref. the RxJS license:

    Copyright (c) 2015-2018 Google, Inc., Netflix, Inc., Microsoft Corp. and contributors

File: LICENSE [hunk 1]
```diff
@@ -1,191 +1,21 @@
...
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
+The MIT License
+===============
+
+Copyright 2013-2019, Dag Bratlli, Microsoft Corp., and Contributors.
```
Comment (dbrattli) [inline LICENSE]: Brattli

File: project.cfg [hunk 2]
```diff
@@ -6,6 +6,6 @@ description = Reactive Extensions (Rx) for Python
 author = Dag Brattli
 author_email = dag@brattli.net
 copyright = 2018-2019, Dag Bratlli
```
[the remark this answers is not in the document]
Comment (dbrattli) [reply project.cfg]: Brattli not Bratlli

**Comment by erikkemperman** [comment]
Ah, I see. Fair enough. Terribly sorry about misspelling your name... Fixed now.

**Comment by erikkemperman** [comment]
Ugh the Travis builds on MacOS are failing all the time on real-time tests of the schedulers. I've been trying to finish up refactoring them but it's not finished yet.

**Comment by jcafhe** [comment]
I was going to approve but I don't feel confident about licensing. Maybe @MainRo is more aware about this?

**Comment by dbrattli** [comment]
Here is btw. the first [announcement](https://blogs.msdn.microsoft.com/interoperability/2012/11/06/ms-open-tech-open-sources-rx-reactive-extensions-a-cure-for-asynchronous-data-streams-in-cloud-programming/) of Microsoft open sourcing Reactive Extensions (Rx). RxPY was developed while I worked for Microsoft and license of RxPY was transferred in 2013 to [CLA-MS-Open-Tech.pdf](https://github.com/ReactiveX/RxPY/files/3190268/CLA-MS-Open-Tech.pdf) in order to make it open source.

Thus I feel we should include MS in the Copyright as "good will", but I don't think we need to ask them for permission to change the license. MS Open Tech does not exist anymore, but was moved back to Microsoft [[1](https://www.zdnet.com/article/microsoft-shutters-its-standalone-open-tech-open-source-subsidiary/)][[2](https://betanews.com/2015/04/17/microsoft-open-technologies-is-closing-is-that-good-or-bad-news-for-open-source/)]. The license of Reactive Extensions was recently moved to the .NET foundation, and the .NET Foundation is [license-agnostic](https://dotnetfoundation.org/about). Projects that come to the Foundation are free to choose any open source license, as defined by the Open Source Initiative.

**Comment by jcafhe** [comment]
Ok so that should be good. Thanks for your insight.

**Comment by erikkemperman** [comment]
Thanks for review!

**Comment by MainRo** [comment]
too late, but anyway: Normally re-licensing requires the agreement of all contributors. Some big projects struggle to do that because it can take a lot of time. However considering the information provided by @dbrattli (the ms license does not exist anymore), and the relatively small number of contributors in rxpy I think this definitely makes sense. Moreover MIT being one of my favorite OSS license, I'm all for it!

**Comment by erikkemperman** [comment]
> Normally re-licensing requires the agreement of all contributors.

Apologies, I guess I was too quick with merging this then. But of course nothing is final, if there are objections things can be changed!

**Pull request merged**